tests/ci/run_bsd_tests.sh

```diff
@@ -41,8 +46,11 @@ build_and_test -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=1
 echo "Testing AWS-LC static library in release mode."
 build_and_test -DCMAKE_BUILD_TYPE=Release
 
-echo "Testing AWS-LC shared library in FIPS Release mode."
-fips_build_and_test -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=1
+# The FIPS builds fail on NetBSD
```

Comment:
Because `OPENSSL_cpucap_initialized` is not set? What the reason it fails?

Reply:
* For the static build it looked like the delocator was unable to parse a specific combination of assembly instruction.
* For the shared build there was a linker error, but I forget the details.